src/proto/Ethereum.proto

```diff
@@ -83,11 +86,20 @@ message SigningInput {
     bytes nonce = 2;
 
     // Gas price (256-bit number)
+    // If > 0, legacy fee scheme is used; if 0, EIP1559 fee scheme is used
     bytes gas_price = 3;
 
     // Gas limit (256-bit number)
     bytes gas_limit = 4;
 
+    // Maxinmum optional inclusion fee (aka tip) (256-bit number)
+    // Used only for EIP1559 fee, disregarded for legacy
+    bytes max_inclusion_fee_per_gas = 8;
+
+    // Maxinmum fee (256-bit number)
+    // Used only for EIP1559 fee, disregarded for legacy
```

Comment:
do you think we can order properties by index? to avoid confusion in the future

Reply:
Not sure what's the best approach:
- change index numbers to match the order --> risk of breaking binary usage (had issues in the past)
- reorder in the source code to match indexes --> confusing, items close logically end up separates
- keep as it is